### PR TITLE
Split formatting guide into multiple pages

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Split the formatting guide into multiples pages (personalise and format). Update links when creating templates, link to these pages in relevant places.

New pages:
- /format
- /personalise
- /guidance

Trello: https://trello.com/c/dbAxE4vw/547-split-formatting-guide-into-multiples-pages

### CODE REVIEW

File: app/templates/views/edit-email-template.html [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
-service_id=current_service.id, template_folder_id=template_folder_id) ) }} {%
-call form_wrapper() %}
-<div class="grid-row contain-floats">
-  <div class="md:w-5/6 float-left py-0 px-0 px-gutterHalf box-border">
-    <div class="form-group contain-floats box-border mb-gutterHalf md:mb-gutter">
-      <a href="{{ url_for('main.templates') }}" target="_blank"
-        >{{ _('How to format your message') }}</a
-      >
-    </div>
+{% extends "admin_template.html" %}
```
Comment (AntoineAugusti) [inline app/templates/views/edit-email-template.html:1]: Reformatted this template

File: tests/app/main/views/test_index.py [hunk 2]
```diff
@@ -111,7 +111,8 @@ def test_security_txt(client):
     'privacy', 'pricing', 'terms', 'roadmap', 'why-notify',
     'features', 'security', 'messages_status',
     'email', 'sms', 'letters', 'welcome',
-    'features',
+    'features', 'format', 'personalise',
+    'guidance',
```
Comment (AntoineAugusti) [inline tests/app/main/views/test_index.py:115]: Making sure these pages return `200 OK`

File: app/main/views/index.py [hunk 3]
```diff
@@ -26,7 +26,6 @@
     FieldWithNoneOption,
     SearchByNameForm,
 )
-from app.main.views.sub_navigation_dictionaries import features_nav
 from app.utils import (
```
Comment (AntoineAugusti) [inline app/main/views/index.py:29]: Removed `features_nav` which was not used/displayed

**Comment by AntoineAugusti** [comment]
Review app: https://notification-formatting-jzppai.herokuapp.com/

**Review by anikbrazeau — APPROVED** [review]
Super! Merci! 🚀  On pourra créer un autre PR pour le lien entre les guides sous peu 😄

**Review by jimleroyer — APPROVED** [review]
:shipit:

**Pull request merged**